docs/postcss.config.js

```diff
@@ -1,5 +1,7 @@
 module.exports = {
   plugins: {
+    'postcss-import': {},
```

Comment:
`postcss-import` is missing in the dependencies?

Reply:
It is already a dependency of tailwindcss which is why it wasn't complaining, but it makes sense to have it directly in the workspace. I've added it - no changes in yarn.lock which is expected.

Reply:
Thanks, my point was no transitive dependencies.